### PR TITLE
Add WebSockets

### PR DESCRIPTION
**Pull request by facundofarias**
https://github.com/facundofarias/awesome-websockets

Resources about WebSockets and its support of different languages.

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- has been around for at least 30 days,
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section named `Table of Contents` as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -468,7 +468,10 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Quantified Self](https://github.com/willempienaar/awesome-quantified-self) - Self-tracking through technology.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
 - [SaltStack](https://github.com/hbokh/awesome-saltstack) - Python-based config management system.
-- [Web Design](https://github.com/nicolesaidy/awesome-web-design) - Resources for digital designers. 
+- [Web Design](https://github.com/nicolesaidy/awesome-web-design) - Resources for digital designers.
```
Comment (davisonio) [inline readme.md]: Seems like you've added some extra items here, might require a rebase

Comment (facundofarias) [inline readme.md]: Sorry, maybe I did something wrong while updating the fork.
What should I do?

Comment (mischah) [inline readme.md]: You could delete the the extra items and push :octocat:

**Comment by sindresorhus** [comment]
> includes a succinct description of the project/theme at the top (example),

Not followed

> doesn't include a Travis badge,

Not followed

> has a Table of Contents section named Table of Contents as the first section,

Not followed

> has an appropriate license (Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc) **(If you use a license badge, it should be SVG, not PNG),**

Not followed

> Drop all the A / An prefixes in the descriptions.

Not followed

> # By submitting this pull request I confirm I've read and complied with the below requirements.

Clearly not...

**Comment by sindresorhus** [comment]
ping

**Comment by facundofarias** [comment]
Hey @sindresorhus, apologies I haven't had the time to solve those issues.
I will take a look this weekend, and will send it again. 
Thanks!

**Comment by sindresorhus** [comment]
ping2 :)

**Comment by facundofarias** [comment]
Hello @sindresorhus, I've created another pull request (#901), please close this one.
Thanks

**Pull request closed without merging**